src/kaleidoscope/hardware/technomancy/Atreus.h

```diff
@@ -65,9 +66,12 @@ class Atreus: public kaleidoscope::hardware::ATMegaKeyboard {
 
   static constexpr int8_t led_count = 0;
 
-  void resetDevice();
+  void resetDevice() {
+    bootloader_.resetDevice();
+  }
```

Comment:
Is there a reason that things that extend ATMegaKeyboard can't use a shared implementation of resetDevice?

Reply:
Yes: Not all of them use the same bootloader. We could extend the `ATMegaKeyboard` macros to also take a bootloader arg, but I don't think that'd be worth it.

What we could do, on the other hand, is this:

```c++
#define KALEIDOSCOPE_USE_BOOTLOADER(x) \
 void resetDevice() { bootloader_.resetDevice(); } \ private: x bootloader_; public:
```